buffer/llbuffer_test.go

```diff
@@ -0,0 +1,401 @@
+package buffer_test
```

Comment:
We use same package name as the original package. Although this pattern is useful for public API testing, we don't do that on s5cmd.